src/strptime.h

```diff
@@ -0,0 +1,50 @@
+
+/*
+ * Polyfill for strptime function
+ *
+ * This source code is from musl library, located at https://git.musl-libc.org/cgit/musl/tree/src/time/strptime.c .
+ */
+
+/*
+ * musl as a whole is licensed under the following standard MIT license:
+ * 
+ * ----------------------------------------------------------------------
+ * Copyright © 2005-2020 Rich Felker, et al.
```

Comment:
Is this copyright header still valid?

Reply:
Nope, I forgot to delete them.
The header file is deleted at d8092b4, resolving this.